extensions/graphcdn/src/search.tsx

```diff
@@ -0,0 +1,270 @@
+import {
+  ActionPanel,
+  List,
+  Detail,
+  useNavigation,
+  showToast,
+  ToastStyle,
+  getPreferenceValues,
+  OpenInBrowserAction,
+  Icon,
+  getLocalStorageItem,
+  setLocalStorageItem,
+} from "@raycast/api";
+import fetch from "node-fetch";
+import { useState, useEffect } from "react";
+
+interface Preferences {
+  token: string;
+}
+
+export default function Command() {
+  const [loading, setLoading] = useState(true);
+  const { push } = useNavigation();
+  const [orgs, setOrgs] = useState<{ id: string; slug: string; name: string }[]>([]);
+  const [services, setServices] = useState<
+    { name: string; updatedAt: number; org: string; stats: { cacheHitRate: number | null } }[]
+  >([]);
+
+  useEffect(() => {
+    async function run() {
+      const cachedOrgs = await getLocalStorageItem("orgs");
+      if (typeof cachedOrgs === "string") {
+        setOrgs(JSON.parse(cachedOrgs));
+      }
+      const cachedServices = await getLocalStorageItem("services");
+      if (typeof cachedServices === "string") {
+        setServices(JSON.parse(cachedServices));
+      }
+
+      const orgs = await getOrgs();
+      setOrgs(orgs);
+
+      const services = (
+        await Promise.all(
+          orgs.map(async (org) => {
+            const services = await getServices(org.slug);
+            console.log(services);
+            return services.map((service) => ({ ...service, org: org.slug }));
+          })
+        )
+      ).flat();
+      setServices(services);
+      setLoading(false);
+
+      await setLocalStorageItem("orgs", JSON.stringify(orgs));
+      await setLocalStorageItem("services", JSON.stringify(services));
+    }
+    run();
+  }, [setOrgs, setServices]);
+
+  const onFetchConfig = async (serviceName: string) => {
+    setLoading(true);
+    try {
+      const config = await getConfig(serviceName);
+      push(
+        <Detail
+          markdown={`\`\`\`\n${JSON.stringify(config, null, "  ")}\n\`\`\``}
+          actions={<ActionPanel></ActionPanel>}
```

Comment:
```suggestion
          actions={<ActionPanel/>}
```
Could be collapsed or removed